### ISSUE TITLE
AirPlay crashes system

### ISSUE THREAD

**Issue by VaBanck** [sle118/squeezelite-esp32#260]
When the track is switched, the device reboots.![image](https://github.com/sle118/squeezelite-esp32/assets/75133860/f20b7cd2-fbd5-45e5-b5e1-35438b0a3abd)

**Comment by sle118** [comment]
There's really nothing I can do to help unless you provide more information or at least a full log. To do so, start by disabling telnet so all the logs are visible in the log window. And please... No screen shot; they cannot be indexed for searching and make it impossible to search. Get the logs, copy paste them and describe exactly what your situation is. There's an infinite way to use the system and we just can't guess.

**Comment by VaBanck** [comment]
> There's really nothing I can do to help unless you provide more information or at least a full log. To do so, start by disabling telnet so all the logs are visible in the log window. And please... No screen shot; they cannot be indexed for searching and make it impossible to search. Get the logs, copy paste them and describe exactly what your situation is. There's an infinite way to use the system and we just can't guess.

**Comment by VaBanck** [comment]
> There's really nothing I can do to help unless you provide more information or at least a full log. To do so, start by disabling telnet so all the logs are visible in the log window. And please... No screen shot; they cannot be indexed for searching and make it impossible to search. Get the logs, copy paste them and describe exactly what your situation is. There's an infinite way to use the system and we just can't guess.

```[00:00:58.630] discover_server:828 sending discovery
[00:01:01.549] rtsp_thread:393 got RTSP connection 54
[00:01:03.630] discover_server:828 sending discovery
[00:01:08.630] discover_server:828 sending discovery
[00:01:08.938] handle_rtsp:447 [0x3fabead8]: challenge perBDI9b5SplknyE1ZqQug==
[00:01:09.222] handle_rtsp:440 [0x3fabead8]: received ANNOUNCE
[00:01:09.496] search_remote:739 starting remote search
[00:01:09.497] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Audio-Jack-Status: connected; type=analog

CSeq: 4




[00:01:09.530] handle_rtsp:440 [0x3fabead8]: received SETUP
[00:01:09.531] raop_sink_cmd_handler:280 resizing buffer 423360
[00:01:09.534] buffer_alloc:386 allocated 1153 buffers (min=375) from buffer of 1624640 bytes
[00:01:09.545] output_thread_i2s:463 Output state is 0
[00:01:09.548] output_thread_i2s:529 Restarting I2S.
[00:01:09.552] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Transport: RTP/AVP/UDP;unicast;mode=record;control_port=56280;timing_port=56281;server_port=56279

Session: DEADBEEF

Audio-Jack-Status: connected; type=analog

CSeq: 5




[00:01:09.553] equalizer_open:71 equalizer initialized 0
[00:01:09.591] handle_rtsp:440 [0x3fabead8]: received RECORD
[00:01:09.592] rtp_record:373 [0x3fac2d54]: record 0 0
[00:01:09.593] raop_sink_cmd_handler:284 Stream
[00:01:09.605] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Audio-Jack-Status: connected; type=analog

CSeq: 6




[00:01:09.637] handle_rtsp:440 [0x3fabead8]: received SET_PARAMETER
[00:01:09.639] handle_rtsp:609 [0x3fabead8]: SET PARAMETER volume -20.000000
[00:01:09.640] raop_sink_cmd_handler:317 Volume[0..1] 0.3333
[00:01:09.652] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Audio-Jack-Status: connected; type=analog

CSeq: 7




[00:01:09.659] rtp_thread_func:708 [0x3fac2d54]: 1st sync packet received
[00:01:09.685] raop_sink_cmd_handler:305 Play
[00:01:09.686] raop_sink_cmd_handler:310 Starting at 39912 (in 2014 ms)
[00:01:09.711] handle_rtsp:440 [0x3fabead8]: received FLUSH
[00:01:09.713] rtp_flush:347 [0x3fac2d54]: FLUSH ignored as same as RECORD (20305 - 1216411328)
[00:01:09.�22] output_thread_i2s:463 Output state is 5
[00:01:09.724] rtp_flush:356 [0x3fac2d54]: flush 20305 1216411328
[00:01:09.727] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Audio-Jack-Status: connected; type=analog

CSeq: 8




[00:01:10.819] handle_rtsp:440 [0x3fabead8]: received SET_PARAMETER
[00:01:10.821] handle_rtsp:638 [0x3fabead8]: received JPEG image of 27653 bytes
[00:01:10.833] handle_rtsp:659 [0x3fabead8]: responding:
RTSP/1.0 200 OK

Audio-Jack-Status: connected; type=analog

CSeq: 9




[00:01:10.879] handle_rtsp:440 [0x3fabead8]: received SET_PARAMETER
[00:01:10.880] handle_rtsp:628 [0x3fabead8]: received metadata
[00:01:10.881] handle_rtsp:633 [0x3fabead8]: received metadata
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x40094d31  PS      : 0x00060030  A0      : 0x802576dc  A1      : 0x3fac0520  
A2      : 0x00000000  A3      : 0xfffffffc  A4      : 0x000000ff  A5      : 0x0000ff00  
A6      : 0x00ff0000  A7      : 0xff000000  A8      : 0x00000000  A9      : 0x3fac04e0  
A10     : 0x00000000  A11     : 0x00000000  A12     : 0x3fac0764  A13     : 0x00000000  
A14     : 0x3f81195c  A15     : 0x00000009  SAR     : 0x00000004  EXCCAUSE: 0x0000001c  
EXCVADDR: 0x00000000  LBEG    : 0x40094d31  LEND    : 0x40094d41  LCOUNT  : 0xffffffff  

Backtrace:0x40094d2e:0x3fac0520 0x402576d9:0x3fac0530 0x40258de5:0x3fac0840 0x40100589:0x3fac0870 0x4010844e:0x3fac08c0 0x40108601:0x3fac0a50


ELF file SHA256: 40f7c3160c93b168

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:6480
load:0x40078000,len:15868
load:0x40080400,len:4384
entry 0x40080684
[0;32mI (27) boot: ESP-IDF v4.3.2-dirty 2nd stage bootloader[0m
[0;32mI (27) boot: compile time 19:21:54[0m
[0;32mI (27) boot: chip revision: 3[0m
[0;32mI (30) boot_comm: chip revision: 3, min. bootloader chip revision: 1[0m
[0;32mI (38) qio_mode: Enabling default flash chip QIO[0m
[0;32mI (43) boot.esp32: SPI Speed      : 80MHz[0m
[0;32mI (48) boot.esp32: SPI Mode       : QIO[0m
[0;32mI (52) boot.esp32: SPI Flash Size : 4MB[0m
[0;32mI (57) boot: Enabling RNG early entropy source...[0m
[0;32mI (62) boot: Partition Table:[0m
[0;32mI (66) boot: ## Label            Usage          Type ST Offset   Length[0m
[0;32mI (73) boot:  0 nvs              WiFi data        01 02 00009000 00004000[0m
[0;32mI (80) boot:  1 otadata          OTA data         01 00 0000d000 00002000[0m
[0;32mI (88) boot:  2 phy_init         RF data          01 01 0000f000 00001000[0m
[0;32mI (95) boot:  3 recovery         factory app      00 00 00010000 00140000[0m
[0;32mI (103) boot:  4 ota_0            OTA app          00 10 00150000 002a0000[0m
[0;32mI (110) boot:  5 settings         WiFi data        01 02 003f0000 00010000[0m
[0;32mI (118) boot: End of partition table[0m
[0;32mI (122) boot_comm: chip revision: 3, min. application chip revision: 0[0m
[0;32mI (129) esp_image: segment 0: paddr=00150020 vaddr=3f400020 size=aed18h (716056) map[0m
[0;32mI (350) esp_image: segment 1: paddr=001fed40 vaddr=3ffbdb60 size=012d8h (  4824) load[0m
[0;32mI (352) esp_image: segment 2: paddr=00200020 vaddr=400d0020 size=1ab994h (1751444) map[0m
[0;32mI (875) esp_image: segment 3: paddr=003ab9bc vaddr=3ffbee38 size=05f8ch ( 24460) load[0m
[0;32mI (883) esp_image: segment 4: paddr=003b1950 vaddr=40080000 size=1db20h (121632) load[0m
[0;32mI (925) esp_image: segment 5: paddr=003cf478 vaddr=400c0000 size=00068h (   104) load[0m
[0;32mI (926) esp_image: segment 6: paddr=003cf4e8 vaddr=50000000 size=00010h (    16) load[0m
[0;32mI (947) boot: Loaded app from partition at offset 0x150000[0m
[0;32mI (947) boot: Disabling RNG early entropy source...[0m
[0;32mI (958) psram: This chip is ESP32-D0WD[0m
[0;32mI (959) spiram: Found 64MBit SPI RAM device[0m
[0;32mI (959) spiram: SPI RAM mode: flash 80m sram 80m[0m
[0;32mI (964) spiram: PSRAM initialized, cache is in low/high (2-core) mode.[0m
[0;32mI (971) cpu_start: Pro cpu up.[0m
[0;32mI (975) cpu_start: Starting app cpu, entry point is 0x40081450[0m
[0;32mI (968) cpu_start: App cpu up.[0m
[0;32mI (1480) spiram: SPI SRAM memory test OK[0m
[0;32mI (1492) cpu_start: Pro cpu start user code[0m
[0;32mI (1492) cpu_start: cpu freq: 240000000[0m
[0;32mI (1492) cpu_start: Application information:[0m
[0;32mI (1495) cpu_start: Project name:     Squeezelite-ESP32[0m
[0;32mI (1501) cpu_start: App version:      I2S-4MFlash.16.2.1306.master-v4[0m
[0;32mI (1508) cpu_start: Compile time:     May 12 2023 19:24:21[0m
[0;32mI (1514) cpu_start: ELF file SHA256:  40f7c3160c93b168...[0m
[0;32mI (1520) cpu_start: ESP-IDF:          v4.3.2-dirty[0m
[0;32mI (1526) heap_init: Initializing. RAM available for dynamic allocation:[0m
[0;32mI (1533) heap_init: At 3FFAFF10 len 000000F0 (0 KiB): DRAM[0m
[0;33mW (1539) REMOVE: NEW POOL of 7268 bytes, ctrl_size: 416 sli_c:8 fli_c:10 small_b 32 max_b:16384[0m
[0;32mI (1549) heap_init: At 3FFB6388 len 00001C78 (7 KiB): DRAM[0m
[0;33mW (1555) REMOVE: NEW POOL of 16628 bytes, ctrl_size: 736 sli_c:16 fli_c:10 small_b 64 max_b:32768[0m
[0;32mI (1564) heap_init: At 3FFB9A20 len 00004108 (16 KiB): DRAM[0m
[0;33mW (1571) REMOVE: NEW POOL of 97132 bytes, ctrl_size: 872 sli_c:16 fli_c:12 small_b 64 max_b:131072[0m
[0;32mI (1580) heap_init: At 3FFC8480 len 00017B80 (94 KiB): DRAM[0m
[0;32mI (1587) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM[0m
[0;32mI (1593) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM[0m
[0;33mW (1599) REMOVE: NEW POOL of 9420 bytes, ctrl_size: 416 sli_c:8 fli_c:10 small_b 32 max_b:16384[0m
[0;32mI (1609) heap_init: At 4009DB20 len 000024E0 (9 KiB): IRAM[0m
[0;32mI (1615) spiram: Adding pool of 4028K of external SPI memory to heap allocator[0m
[0;33mW (1623) REMOVE: NEW POOL of 4124667 bytes, ctrl_size: 2168 sli_c:32 fli_c:16 small_b 128 max_b:4194304[0m
[0;32mI (1634) spi_flash: detected chip: generic[0m
[0;32mI (1638) spi_flash: flash io: qio[0m
[0;33mW (1642) spi_flash: Detected size(16384k) larger than the size in the binary image header(4096k). Using the size in the binary image header.[0m
[0;32mI (1658) cpu_start: Starting scheduler on PRO CPU.[0m
[0;32mI (0) cpu_start: Starting scheduler on APP CPU.[0m
[0;33mW (1666) REMOVE: NEW POOL of 15052 bytes, ctrl_size: 416 sli_c:8 fli_c:10 small_b 32 max_b:16384[0m
[0;33mW (1676) REMOVE: NEW POOL of 113820 bytes, ctrl_size: 872 sli_c:16 fli_c:12 small_b 64 max_b:131072[0m
[0;32mI (1686) spiram: Reserving pool of 64K of internal memory for DMA/internal allocations[0m
[0;33mW (1696) REMOVE: NEW POOL of 65515 bytes, ctrl_size: 804 sli_c:16 fli_c:11 small_b 64 max_b:65536[0m
[0;32mI (1706) esp_app_main: Reset reason is: 4[0m
[0;32mI (1706) esp_app_main: Reboot counter=1
[0m
[0;32mI (1716) esp_app_main: Starting app_main[0m
[0;32mI (1716) nvs_utilities: Initializing flash nvs [0m
[0;32mI (1736) nvs_utilities: Initializing nvs partition settings[0m
[0;32mI (1776) esp_app_main: Setting up telnet.[0m
[0;32mI (1776) telnet: Telnet support disabled[0m
[0;32mI (1776) esp_app_main: Setting up config subsystem.[0m
[0;33mW (1836) nvs_utilities: Configuration memory usage.  Heap internal:224899 (min:224735) (used:200) external:4107495 (min:4100135) (used:12724)[0m
[0;32mI (1846) esp_app_main: Registering default values[0m
[0;32mI (1846) system_api: Base MAC address is not set[0m
[0;32mI (1856) system_api: read default base MAC address from EFUSE[0m
[0;33mW (1856) config: Waiting for config commit ...[0m
[0;32mI (2846) config: configuration has some uncommitted entries[0m
[0;32mI (2846) config: Committing configuration to nvs. Locking config object.[0m
[0;32mI (2846) config: Done Committing configuration to nvs.[0m
[0;32mI (2846) config: Config committed![0m
[0;32mI (2856) esp_app_main: Configuring services[0m
[0;32mI (2856) services: Configuring I2C sda:-1 scl:-1 port:1 speed:400000[0m
[0;33mW (2866) services: no I2C configured[0m
[0;32mI (2866) services: Configuring SPI mosi:-1 miso:-1 clk:-1 host:1 dc:-1[0m
[0;33mW (2876) services: no SPI configured[0m
[0;33mW (2886) led: LED GPIO -1 ignored[0m
[0;33mW (2886) led: LED GPIO -1 ignored[0m
[0;32mI (2886) led: Configuring LEDs green:-1 (active:0 -1%), red:-1 (active:0 -1%)[0m
[0;32mI (2896) battery: No battery[0m
[0;32mI (2896) monitor: Initializing monitoring[0m
[0;32mI (2906) monitor: Heap internal:225691 (min:224735) external:4091015 (min:4090995) dma:216687 (min:215743)[0m
[0;32mI (2916) esp_app_main: Initializing display[0m
[0;32mI (2916) display: Trying to configure display with N/A[0m
[0;33mW (2926) display: No display driver[0m
[0;32mI (2926) esp_app_main: Checking for update url[0m
[0;33mW (2936) config: Value not found for key f[0;33mW (2946) config: Value not found for key apdelay[0m
[0;32mI (2946) uart: queue free spaces: 3[0m
[0;32mI (2956) network_handlers: Network manager configuration: polling max 10176, polling min 15000, ap delay 20000, dhcp timeout 8000, eth timeout 8000[0m
[0;32mI (2976) http_server: Initializing HTTP Server[0m
[0;32mI (2986) network_ethernet: Attempting to initialize Ethernet[0m
[0;32mI (2986) network_ethernet: No Ethernet configuration, or configuration invalid[0m
I (3006) wifi:wifi driver task: 3ffcc120, prio:23, stack:6656, core=0
I (3006) wifi:wifi firmware version: eb52264
I (3006) wifi:wifi certification version: v7.0
I (3006) wifi:config NVS flash: enabled
I (3016) wifi:config nano formating: disabled
I (3016) wifi:Init data frame dynamic rx buffer num: 40
I (3016) wifi:Init management frame dynamic rx buffer num: 40
I (3026) wifi:Init management short buffer num: 32
I (3026) wifi:Init static tx buffer num: 12
I (3036) wifi:Init tx cache buffer num: 32
I (3036) wifi:Init static rx buffer size: 1600
I (3046) wifi:Init static rx buffer num: 12
I (3056) wifi:Init dynamic rx buffer num: 40
[0;32mI (3056) wifi_init: tcpip mbox: 32[0m
[0;32mI (3056) wifi_init: udp mbox: 32[0m
[0;32mI (3066) wifi_init: tcp mbox: 32[0m
[0;32mI (3066) wifi_init: tcp tx win: 8192[0m
[0;32mI (3066) wifi_init: tcp rx win: 32768[0m
[0;32mI (3076) wifi_init: tcp mss: 1440[0m
[0;32mI (3076) wifi_init: WiFi/LWIP prefer SPIRAM[0m
[0;32mI (3076) network_wifi: Starting wifi[0m
[0;32mI (3086) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07[0m
I (3176) wifi:mode : sta (c0:49:ef:93:38:40)
I (3176) wifi:enable tsf

[0;32mI (3186) network_handlers: Existing wifi config found. Attempting to connect.[0m
[0;32mI (3196) network: Starting DHCP client[0m
[0;32mI (3196) network_wifi: Wifi Connecting to Home-2...[0m
Type 'help' to get the list of commands.
Use UP/DOWN arrows to navigate through command history.
Press TAB when typing command name to auto-complete.

To automatically execute lines at startup:
	Set NVS variable autoexec (U8) = 1 to enable, 0 to disable automatic execution.
	Set NVS variable autoexec[1~9] (string)to a command that should be executed automatically
```

**Comment by VaBanck** [comment]
iPhone 11, airplay
I listen to the radio when I switch to another station, the device reboots, on some radio stations everything works well, on some I don't want to work

And when the stream from Apple TV (e.g. YouTube) is turned on, the sound takes only 10 seconds

**Comment by philippe44** [comment]
Can you identify with which version the problem started? Use a version prior to mid-March

**Comment by VaBanck** [comment]
> Can you identify with which version the problem started? Use a version prior to mid-March 

2.1288-16#v4.3#12S-4MFlash# master-v4.3, works well before this version, installed version 1271

**Comment by philippe44** [comment]
Sorry this is a bit confusing. 1288 works or not?

**Comment by VaBanck** [comment]
1288 doesn't work

**Comment by philippe44** [comment]
I can't reproduce it yet. Do you have buttons and/or a display? Can you share your config?

**Comment by philippe44** [comment]
Now, I just had an idea and pushed a change. It will not be an official build, so please get the latest builds artifacts (under Actions) and do a local update

**Comment by VaBanck** [comment]
> Now, I just had an idea and pushed a change. It will not be an official build, so please get the latest builds artifacts (under Actions) and do a local update

There is no display and encoder, only i2s dac. Installed the latest version (local files) did not help

**Comment by VaBanck** [comment]
```{
  "a2dp_ctmt": "1000",
  "a2dp_ctrld": "500",
  "a2dp_dev_name": "HomePod",
  "a2dp_sink_name": "SMSL BT4.2",
  "a2dp_spin": "0000",
  "actrls_config": "",
  "airplay_name": "HomePod",
  "airplay_port": "5000",
  "ap_channel": "1",
  "ap_ip_address": "192.168.4.1",
  "ap_ip_gateway": "192.168.4.1",
  "ap_ip_netmask": "255.255.255.0",
  "ap_pwd": "squeezelite",
  "ap_ssid": "HomePod",
  "autoexec": "1",
  "autoexec1": "squeezelite -o I2S -b 500:2000 -d all=info -C 30 -W -n squeezelite-933840",
  "bat_config": "",
  "bt_name": "HomePod-bl",
  "bt_sink_pin": "1234",
  "bt_sink_volume": "127",
  "bypass_wm": "0",
  "cspot_config": "{\"deviceName\":\"HomePod\",\"bitrate\":320}",
  "dac_config": "model=I2S,bck=26,ws=25,do=32",
  "dac_controlset": "",
  "dhcp_tmout": "8",
  "display_config": "",
  "enable_airplay": "Y",
  "enable_bt_sink": "N",
  "enable_cspot": "N",
  "equalizer": "0,0,0,0,0,0,0,0,0,0",
  "eth_config": "",
  "ethtmout": "8",
  "gpio_exp_config": "",
  "host_name": "HomePod",
  "i2c_config": "",
  "jack_mutes_amp": "n",
  "led_brightness": "",
  "lms_ctrls_raw": "n",
  "metadata_config": "",
  "model_config": "",
  "ota_erase_blk": "249856",
  "ota_prio": "6",
  "ota_stack": "10240",
  "pollmin": "15",
  "pollmx": "600",
  "rel_api": "https://api.github.com/repos/sle118/squeezelite-esp32/releases",
  "release_url": "https://github.com/sle118/squeezelite-esp32/releases",
  "rotary_config": "",
  "set_GPIO": "",
  "spdif_config": "",
  "spi_config": "",
  "stats": "N",
  "target": "",
  "telnet_block": "500",
  "telnet_buffer": "40000",
  "telnet_enable": "N"
}```

**Comment by philippe44** [comment]
Crap - so I've undone some mods of 1288 to see. That should be building while I'm writing this, under the "enable DMPA_FULL" tag. Let me know if this changes something because it really seems to be linked to receiving some metadata but I'm shooting in the dark as I can't reproduce and I can't figure out from re-reading the code, so it's trial and error

**Comment by VaBanck** [comment]
> Crap - so I've undone some mods of 1288 to see. That should be building while I'm writing this, under the "enable DMPA_FULL" tag. Let me know if this changes something because it really seems to be linked to receiving some metadata but I'm shooting in the dark as I can't reproduce and I can't figure out from re-reading the code, so it's trial and error

Do I need to download the latest version from the repository And install it manually, am I doing the right thing?

**Comment by philippe44** [comment]
You have to take this file https://github.com/sle118/squeezelite-esp32/suites/13330870751/artifacts/728549108, unzip-it and then in the WebUI of your SqueezeESP32, "Update" tab, select "Local File" and pick the "squeezelite.bin" file that was just extracted from the above mentioned archive.

NB: The link I've picked is the build I've made earlier today, the latest one was still under construct but the files will be in the "Action" tab of GitHub, click on the latest build and then scroll till you find "artifacts"

**Comment by VaBanck** [comment]
Now it doesn't work at all.

**Comment by philippe44** [comment]
> Now it doesn't work at all.
>
What do you mean? I just downloaded the SqueezeAMP version on my board and it works as usual. Did you select the squeezelite.bin file (not the full zip)?

Also does it crash when you use iTunes?

**Comment by VaBanck** [comment]
I'll leave version 1271, it works, maybe it will be better in the future, tomorrow I'll make a log, yes, I installed squeezelit.bin

I'm trying the radio via the Internet

**Comment by sle118** [comment]
@VaBanck there was a recent change done around the vorbis component (https://github.com/sle118/squeezelite-esp32/commit/8b764c0c2d7502f4f76ab23293d1e6ce9ebcc898) you might want to give it another shot by installing the following binary from our automated build (as the changes were not yet made available through a release): https://github.com/sle118/squeezelite-esp32/suites/13388676178/artifacts/732857373

**Comment by VaBanck** [comment]
> @VaBanck there was a recent change done around the vorbis component (https://github.com/sle118/squeezelite-esp32/commit/8b764c0c2d7502f4f76ab23293d1e6ce9ebcc898) you might want to give it another shot by installing the following binary from our automated build (as the changes were not yet made available through a release): https://github.com/sle118/squeezelite-esp32/suites/13388676178/artifacts/732857373

This version doesn't work for me, I'll try another esp, maybe it will work

**Comment by VaBanck** [comment]
> zip)?

Excuse me please, I installed the wrong version, version 1319 works,![image](https://github.com/sle118/squeezelite-esp32/assets/75133860/9fa82c6b-1503-489d-a085-cd09ee410c59)

**Comment by VaBanck** [comment]
https://github.com/sle118/squeezelite-esp32/suites/13337862394/artifacts/729093462

**Comment by VaBanck** [comment]
YouTube app on AppleTV, the sound only plays 10 seconds

**Comment by philippe44** [comment]
I'm confused - can you summarize the situation?

**Comment by VaBanck** [comment]
> I'm confused - can you summarize the situation?

https://github.com/sle118/squeezelite-esp32/suites/13337862394/artifacts/729093462

That version works.

**Comment by philippe44** [comment]
This is more than weird because I've not changed anything compared to the previous version where you told me it would not start. Are you 100000% sure you took the artifacts from the previous version? If not, I'll undo the metadata change and to an official release so that we can compare

**Comment by philippe44** [comment]
ok, I'll consider it closed then
